core/lib/jobs/jobs/jobs.py

```diff
@@ -219,28 +219,66 @@ def _get_placeholders_str(num_placeholders, start=1):
     return ", ".join(f"${i}" for i in range(start, start + num_placeholders))
 
 
+async def get_customer_quota(con, customer_id, service) -> Dict[str, int]:
+    """Query DB and return usage limit and current usage.
+    Returns:
+        - Dictionary with account limits and account usage
+    """
+    # get service specific usage restrictions for the customer account
+    # uploads limit, jobs limit, end date of plan
+    usage_limit_query = "SELECT usage_restrictions->$1 AS usage FROM customers WHERE id=$2"
+    # get total uploads of customer for specific service
+    current_usage_query = "SELECT COUNT(*) as total_uploads, SUM(jobs_count) as total_jobs FROM (SELECT COUNT(*) AS jobs_count FROM jobs_result WHERE customer_id=$1 AND type=$2 GROUP BY upload_id) dt"
+    # get all uploads with up to 2 jobs
+    less_than_3_jobs_query = "SELECT upload_id, COUNT(upload_id) FROM jobs_result WHERE customer_id=$1 GROUP BY upload_id HAVING COUNT(upload_id)<=2"
```

Comment:
We might want to also make sure that we only select from jobs for a specific product type like in the query on 231.. I haven't checked with product but I'm assuming there will be limits per product type.